R/post_processing_functions.R

```diff
@@ -60,7 +60,7 @@ heatPlot_fixed <-function(mat,t_rank){
     }
 
   par(mfrow=c(1,1))
-  image.plot(mat,col=tim.colors(64*10),axes=F, cex.lab=1.5, zlim=range(0,1),
+  fields::image.plot(mat,col=fields::tim.colors(64*10),axes=F, cex.lab=1.5, zlim=range(0,1),

```

Comment:
Let's hope that the functions used from the `fields`package can be replaced by ggplot2 eventually. Very fine for now.